### PR TITLE
FXIOS-717 ⁃ Adds new feature bookmark all tabs

### PR DESCRIPTION
**Pull request by elemanhillary**
Closes [#6110](https://github.com/mozilla-mobile/firefox-ios/issues/6110)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-717)

### CODE REVIEW

File: XCUITests/FxScreenGraph.swift [hunk 1]
```diff
@@ -173,6 +173,7 @@ class Action {
     static let SelectTopSitesRows = "SelectTopSitesRows"
 
     static let GoToHomePage = "GoToHomePage"
+    static let GotoHomePage = "BookmarkAll"
```
Comment (Vest) [inline XCUITests/FxScreenGraph.swift:169]: I am not a real reviewer, but isn't here a typo? Should the line be:
```swift
static let BookmarkAll= "BookmarkAll"
```

**Comment by isabelrios** [comment]
oh @elemanhillary I'm really sorry I missed the review... I will take a look at this PR as soon as I can!

File: XCUITests/FxScreenGraph.swift (hunk 1 above)
Comment (elemanhillary) [reply XCUITests/FxScreenGraph.swift:169]: yeah sure

**Comment by isabelrios** [comment]
@elemanhillary sorry I thought this was only about adding tests, but I see this PR would add a new feature. Let me ask our PM about this.
Hi @athomasmoz! is this a feature that we discussed to be added? sorry if I missed that...before reviewing wanted to confirm this is something we would add. 
Thanks!

**Comment by Nexuist** [comment]
Hi all,

Sorry if I’ve missed something, but was there any more progress on this issue since April? Was this feature ever added or was it decided to not do it?

Regardless, is there anything I can do to help?

Sincerely,
Firefox iOS user with >500 open tabs 😅

**Comment by elemanhillary** [comment]
@Nexuist the feature was done by me, i finished it, it was not added, as you can see the PR is open

**Comment by Nexuist** [comment]
@elemanhillary I'm not familiar with how Mozilla handles PRs; is there anyone we can @ or email in order to figure out what the hold up is? Does Mozilla just not want to add this feature at all for some reason?

**Comment by elemanhillary** [comment]
@Nexuist lets sit and wait

**Comment by athomasmoz** [comment]
Hi @Nexuist and @elemanhillary please see the latest comment here https://github.com/mozilla-mobile/firefox-ios/issues/6110#issuecomment-711481724

**Comment by dnarcese** [comment]
Closing this for now as it needs some UX input before we can merge.
@elemanhillary Thanks for your contribution. When we receive UX support, feel free to pick this issue up again!

**Pull request closed without merging**